docs/public-networks/reference/cli/options.md

```diff
@@ -3302,34 +3302,41 @@ Use the [`miner_changeTargetGasLimit`](../api/index.md#miner_changetargetgaslimi
 the `target-gas-limit` while Besu is running. Alternatively restart Besu with an updated
 `target-gas-limit` value.
 
-### `tx-pool-future-max-by-account`
+### `tx-pool-limit-by-account-percentage`
 
 === "Syntax"
 
     ```bash
-    --tx-pool-future-max-by-account=<INTEGER>
+    --tx-pool-limit-by-account-percentage=<LONG>
     ```
 
 === "Example"
 
     ```bash
-    --tx-pool-future-max-by-account=100
+    --tx-pool-limit-by-account-percentage=0.1
     ```
 
 === "Environment variable"
 
     ```bash
-    BESU_TX_POOL_FUTURE_MAX_BY_ACCOUNT=100
+    BESU_TX_POOL_LIMIT_BY_ACCOUNT_PERCENTAGE=0.1
     ```
 
 === "Configuration file"
 
     ```bash
-    tx-pool-future-max-by-account="100"
+    tx-pool-limit-by-account-percentage=0.4
     ```
 
-The maximum number of future transactions kept in the transaction pool, per account.
-The default is 64.
+The maximum percentage of future transactions kept in the transaction pool, per account.
+The default is .1 or 10% of transactions from a single account to be kept in the pool.
```

Comment:
The current default is .001 or 0.1%, that for the default size means max 5 txs per sender